src/Microsoft.OData.Client/PublicAPI/net8.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,1030 @@
+Microsoft.OData.Client.UriFunctionAttribute

```

Comment:
How come your comment here https://github.com/OData/odata.net/pull/2851/files#r1482628586 does apply to this Shipped.txt file?

Reply:
See: https://github.com/OData/odata.net/pull/2851#discussion_r1484004611